hudi-flink/src/main/java/org/apache/hudi/operator/FlinkOptions.java

```diff
@@ -81,6 +81,12 @@ private FlinkOptions() {
       .noDefaultValue()
       .withDescription("Avro schema file path, the parsed schema is used for deserializing");
 
+  public static final ConfigOption<String> READ_SCHEMA_STRING = ConfigOptions
+      .key("read.schema.string")
+      .stringType()
+      .noDefaultValue()
+      .withDescription("Avro schema string, the parsed schema is used for deserialization");
```

Comment:
Thinking if we'd better add the `AVRO` keyword into the variable name.

Reply:
Yes, rename to `read.avro-schema`, remove the string suffix because apparently it is.